backend/api/simulation.py

```
@@ -3,7 +3,7 @@
 Handles guided simulation with AI personas, goal validation, and progress tracking
 """
 
-from fastapi import APIRouter, Depends, HTTPException, status
+from fastapi import APIRouter, Depends, HTTPException, status, Query
```

Ruff (0.12.2)

`fastapi.status` imported but unused

Remove unused import: `fastapi.status`

(F401)